lab/src/Microsoft.SyndicationFeed/src/Atom/AtomFeedFormatter.cs

```diff
@@ -99,15 +109,15 @@ public bool TryParseValue<T>(string value, out T result)
 
             //
             // DateTimeOffset
-            //if (type == typeof(DateTimeOffset))
-            //{
-            //    DateTimeOffset dt;
-            //    if (DateTimeUtils.TryParse(value, out dt))
-            //    {
-            //        result = (T)(object)dt;
-            //        return true;
-            //    }
-            //}
+            if (type == typeof(DateTimeOffset))
+            {
+                DateTimeOffset dt;
+                if (DateTimeUtils.TryParseAtom(value, out dt))
```

Comment:
Remove these functions from DateTimeUtils and place them into their respective formatters. All utility classes should have only common logic, not specific for a given protocol.